Cloud Application Manager/API/centurylink-private-node.md

```diff
@@ -0,0 +1,54 @@
+{{{ "title": "CenturyLink Private Node",
```

Comment:
Just wondering out loud.....the private node has no relationship to CAM nor is CAM noted in this kb....so why live inside CAM / API area of kb?  will make it pretty hard for a person to find via browsing.

the other comment is the details about the user experience and API but then it goes into a full product description that really doesn't fit the API category at all.  are we trying to jam an entire product overview into an API kb area?  

just feels a bit out of place that could be hard to find.  maybe a private node kb area is better.  pretty easy to do.  @lmcornejo @gavin-lai what's your view?